docs/book/how-do-i-write-policies.md

```diff
@@ -1161,7 +1171,7 @@ following syntax:
 ```
 
 The `<target>`s must be references to values in the input document (or the input
-document itself). The `<value>`s may be Scalar Values, Variables, or Composite
+document itself) or data document. The `<value>`s may be Scalar Values, Variables, or Composite
```

Comment:
We can remove this last sentence now. It was added in the past because we couldn't have with modifiers with refs as values.

Reply:
Done